src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -104,60 +104,48 @@ export const DatasetSearchTable = (props) => {
       queryChunks.push(...searchModifier);
     }
 
-    var filterQuery = {};
+    let filterQuery = {};
     if (numSelectedFilters(filters) > 0) {
-      const shouldTerms = [];
+      const filterTerms = [];
 
       filters.accessManagement.forEach(term => {
-        shouldTerms.push({
+        filterTerms.push({
           'term': {
             'accessManagement': term
           }
         });
       });
 
       filters.dataUse.forEach(term => {
-        shouldTerms.push({
+        filterTerms.push({
           'match': {
             'dataUse.primary.code': term
           }
         });
       });
 
-      if (shouldTerms.length > 0) {
+      if (filterTerms.length > 0) {
         filterQuery = [
           {
             'bool': {
-              'should': shouldTerms
+              'must': filterTerms
```

Comment:
That's a good point. I noticed this while testing, but wasn't sure if the filter categories were always exclusionary, e.g. what about studies that have both open and controlled access data, or have all controlled access data but contain datasets with different data use codes?

OTOH I agree that I'd expect a filter on an e-commerce site to work in this way so that alone seems like an argument to use OR within a filter group.

Reply:
I've updated the code to use OR within a filter group